src/Modules/Backend/Domain/User/Command/ChangeUserHandler.php

```diff
@@ -6,6 +6,7 @@
 use ForkCMS\Modules\Backend\Domain\User\Event\UserChangedEvent;
 use ForkCMS\Modules\Backend\Domain\User\User;
 use ForkCMS\Modules\Backend\Domain\User\UserRepository;
+use Scheb\TwoFactorBundle\Security\TwoFactor\Provider\Google\GoogleAuthenticatorInterface;
```

Comment:
Is this actually used?

Reply:
fixed 3ff93635850f4a226736fb89cc315e9fb92c9ea1